tests/conftest.py

```diff
@@ -35,6 +44,13 @@
 # --- Pytest config --------------------------------------------------------------------
 
 
+def check_webdrivers():
+    for driver in WEBDRIVERS.values():
+        if shutil.which(driver):
+            return True
+    return False
```

Comment:
This should be the `splinter_webdriver` fixture. Change body to:

```python
@pytest.fixture(scope='session')
def splinter_webdriver():
    for driver, executable in WEBDRIVERS.items():
        if shutil.which(executable):
            return driver
    pytest.skip("No webdrivers available")
```

Next task: make this compatible with a CLI override using `pytest --splinter-webdriver=whatever`.